src/backend/access/aocs/aocsam.c

```diff
@@ -1268,7 +1298,11 @@ aocs_gettuple(AOCSScanDesc scan, int64 targrow, TupleTableSlot *slot)
 				/* continue next block */
 			}
 			else
-				pg_unreachable(); /* unreachable code */
+				ereport(ERROR,
+						(errcode(ERRCODE_INTERNAL_ERROR),
+						 errmsg("Unexpected EOF during analysis of AOCO table \'%s\' for column \'%s\'",
```

Comment:
I think that string without `\` should work as well
```suggestion
						 errmsg("Unexpected EOF during analysis of AOCO table '%s' for column '%s'",
```